Examples/Scripts/PostFreeSurferPipelineLongPrepBatch.sh

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash 
```

Comment:
I don't understand the purpose of this script if it is included above.  It also has 0.7mm instead of 0.8mm templates.

Reply:
This is a dev version for debugging purposes. I'd like to keep it with the main repo for now as it gets increasingly difficult to track/reuse over time if I just keep a local copy. Can we put it (and similar scripts) into a separate silo named something like 'dev' or 'debug'?

Reply:
We haven't done anything like that before.  If it is in a separate branch on the same repo, it probably won't be any easier to keep up to date (checking out that branch would revert any commits to the script it is launching, so you'd need to rebase and edit, then commit, which breaks the link between the commit in the dev branch and the commit to master it works with).  We do have some old example scripts moved into a "Historical" folder, so we might consider another folder for dev testing scripts...

I tend to use a separate clone of the repo for unrelated nontrivial projects, and keep my testing scripts in a folder beside the repo.  I have never needed to edit them to match an older commit's launch parameters, and git can't edit the example to match new changes you have made to the pipeline, so I'm not entirely clear what the benefit of putting them in the repo would be (expectation of losing track of the filesystem you were working on?  not having the same filesystem available in different work locations?).

Reply:
Noted. At this point, batch and launcher scripts changed so much that it doesn't make sense to keep the PostFSLongPrep which I haven't updated anyways so I'm removing it from the repo.